### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by aschampion** [PistonDevelopers/image#743]
It's been quite some time since the last release (4 months compared to a median inter-release interval around 2 months). For those of us distributing crates depending on `image`, a new release would be useful if we're blocking on new features and bugfixes. Even better if a resolution to the Rayon dependency issue (#723/#724) could land beforehand.

Would a version/doc bumping PR help for this to take some maintenance workload off the maintainers? Relatedly, is `image` looking for more maintainers?

**Comment by bvssvni** [comment]
Yeah, we're looking for more maintainers.

We started https://gitter.im/PistonDevelopers/rust-image-group to get people interested in image libraries together. I suggested that people making PRs review each other's PRs to help with the burden of maintenance.

I plan making a new release soon, but there is bug in the inflate crate that I'm waiting for to get sorted out. People are not yet back from Easter holiday. Bad timing I guess.
